package.json

```diff
@@ -62,15 +62,15 @@
     "rollup": "^2.34.2"
   },
   "peerDependencies": {
-    "react": ">=16.8.0",
-    "react-dom": ">=16.8.0"
+    "react": "^16.14 || ^17.0",
+    "react-dom": "^16.14 || ^17.0"
```

Comment:
Made these a little bit more specific, just in case our package doesn't work with React 18 in the future.